homework/test.js

```diff
@@ -0,0 +1,56 @@
+import { expect } from 'chai';
+import {  ShopError, Aliments, Furniture, Customer, addProductTest, fundsTest, emptyCartTest, paymentTest } from './homework.js';
+
+describe('AddAliment', () => {
+    it('Valid Aliment', () => {
+        var futureDate = new Date(); 
+        futureDate.setDate(new Date() + 1);
```

Comment:
`new Date() + 1` converts the Date instance to its string representation and then concatenates it with the other operand.

You should use `Date.now()` if you want to operate on UNIX timestamps and make sure arithmetic works fine.